### PR TITLE
feat(eslint): one import per line + import alphabetize + export all alphabetize

### PR DESCRIPTION
**Pull request by matthieu-crouzet**
## Proposed change
One import per line
Import alphabetize
Export all alphabetize

### CODE REVIEW

File: packages/@o3r/eslint-config/schematics/ng-add/vscode/index.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
-import type { JsonObject } from '@angular-devkit/core';
-import { chain, type Rule } from '@angular-devkit/schematics';
+import type {
```
Comment (kpanot) [inline packages/@o3r/eslint-config/schematics/ng-add/vscode/index.ts:1]: 😢

File: packages/@o3r/eslint-config/schematics/ng-add/vscode/index.ts [hunk 2]
```diff
@@ -13,6 +18,7 @@ export const updateVscode: Rule = async () => {
       const settings = tree.exists(vscodeSettingsPath) ? (tree.readJson(vscodeSettingsPath) || {}) as JsonObject : {};
       settings['eslint.useFlatConfig'] = true;
       settings['editor.defaultFormatter'] = 'dbaeumer.vscode-eslint';
+      settings['editor.foldingImportsByDefault'] = true;
```
Comment (matthieu-crouzet) [inline packages/@o3r/eslint-config/schematics/ng-add/vscode/index.ts:21]: Thanks to `unused-imports/no-unused-imports` we can collapse all imports by default :)

**Review by kpanot — APPROVED** [review]

File: packages/@o3r/eslint-config/src/rules/typescript/import.cjs [hunk 3]
```diff
@@ -0,0 +1,25 @@
+/**
+ * @type {import('@typescript-eslint/utils').TSESLint.FlatConfig.ConfigArray}
+ */
+const config = [
+  {
+    name: '@o3r/overrides/import',
+    files: [
+      '**/*.{c,m,}{t,j}s'
+    ],
+    rules: {
```
Comment (mrednic-1A) [inline packages/@o3r/eslint-config/src/rules/typescript/import.cjs:10]: We did not talked about it, but for imports I would add a trailing comma mandatory.
In this way if a new import is added at the end of the list only one line will appear on the diff.
Without trailing comma the diff will be on 2 lines.
https://eslint.style/rules/js/comma-dangle#options

File: eslint.shared.config.mjs [hunk 4]
```diff
@@ -1,10 +1,9 @@
-import globals from 'globals';
+import o3rTemplate from '@o3r/eslint-config/template';
```
Comment (matthieu-crouzet) [inline eslint.shared.config.mjs:1]: I don't understand why it does not go next to `import o3rConfig from '@o3r/eslint-config';` but if it is automatic now we don't really need to review the order, everyone will have the same

**Review by mrednic-1A — APPROVED** [review]

**Pull request merged**